tr/cs-221-logic-models.md

```diff
@@ -0,0 +1,462 @@
+
+<br>
+
+
+**49. Substitution ― A substitution θ maps variables to terms and Subst[θ,f] denotes the result of substitution θ on f.**
+
+&#10230; Yer değiştirme - Bir yerdeğiştirme değişkenleri terimlerle eşler ve Subst[θ,f] yerdeğiştirme sonucunu f olarak belirtir.
+
+<br>
+
+
+**50. Unification ― Unification takes two formulas f and g and returns the most general substitution θ that makes them equal:**
+
+&#10230; Birleştirme ― Birleştirme f ve g'nin iki formülünü alır ve onları eşit yapan en genel ikameyi θ verir:
+
+<br>
+
+
+**51. such that**
+
+&#10230; öyle ki
+
+<br>
+
+
+**52. Note: Unify[f,g] returns Fail if no such θ exists.**
+
+&#10230; Not: Unify[f,g], eğer böyle bir θ yoksa Fail döndürür.
+
+<br>
+
+
+**53. Modus ponens ― By noting x1,...,xn variables, a1,...,ak and a′1,...,a′k atomic formulas and by calling θ=Unify(a′1∧...∧a′k,a1∧...∧ak) the first-order logic version of modus ponens can be written:**
+
+&#10230; Modus ponens ― x1, ..., xn değişkenleri, a1, ..., ak ve a′1, ..., a′k atomik formüllerine dikkat ederek ve θ=Unify(a′1∧...∧a′k,a1∧...∧ak) modus ponenlerin birinci dereceden mantık versiyonu yazılabilir:
+
+<br>
+
+
+**54. Completeness ― Modus ponens is complete for first-order logic with only Horn clauses.**
+
+&#10230; Tamlık - Modus ponens sadece Horn cümleleriyle birinci dereceden mantık için tamamlanmıştır.
+
+<br>
+
+
+**55. Resolution rule ― By noting f1,...,fn, g1,...,gm, p, q formulas and by calling θ=Unify(p,q), the first-order logic version of the resolution rule can be written:**
+
+&#10230; Çözünürlük kuralı ― f1,...,fn,g1,...,gm, p, q formüllerini not ederek ve θ=Unify(p,q) ifadesini kullanarak, çözümleme kuralının birinci dereceden mantık sürümü yazılabilir. :
+
+<br>
+
+
+**56. [Semi-decidability ― First-order logic, even restricted to only Horn clauses, is semi-decidable., if KB⊨f, forward inference on complete inference rules will prove f in finite time, if KB⊭f, no algorithm can show this in finite time]**
+
+&#10230; Yarı karar verilebilirlik ― Birinci dereceden mantık, sadece Horn cümleleriyle sınırlı olsa bile,  yarı kararsıdır eğer KB⊨f ise f sonsuz zamanlıdır. KB⊭f ise sonsuz zamanlı olabilirliği gösteren algoritma yoktur.
+
+<br>
+
+
+**57. [Basics, Notations, Model, Interpretation function, Set of models]**
+
+&#10230; [Temeller, Notasyon, Model, Yorumlama fonksiyonu, Modellerin kümesi]
+
+<br>
+
+
+**58. [Knowledge base, Definition, Probabilistic interpretation, Satisfiability, Relationship with formulas, Forward inference, Rule properties]**
+
+&#10230;  [Bilgi temeli, Tanım, Olasılıksal yorumlama, Gerçeklenebilirlik, Formüllerle İlişki, İleri çıkarım, Kural özellikleri]
+
+<br>
+
+
+**59. [Propositional logic, Clauses, Modus ponens, Conjunctive normal form, Representation equivalence, Resolution]**
+
+&#10230; [Önerme mantığı, Cümleler, Modus ponens, Eşlenik (Conjunctive) normal form, Temsil eşdeğeri, Çözüm]
```

Comment:
Resolution için hep çözünürlük kullanıldı daha önceki kısımlarda

Reply:
ben neden çözüm yazdım acaba zaten tabi ki çözünürlük :)